### ISSUE TITLE
Wiki verfollständigen: System-Check: Netzwerkprüfung

### ISSUE THREAD

**Issue by franikowsp** [iqb-berlin/testcenter#397]
Ebenfalls nochmal den System-Check betreffend (daher leider wieder dringend). Ist es entweder möglich, diese Message komplett auszublenden:
>"Die folgenden Netzwerkeigenschaften wurden festgestellt: Ihre Verbindung zum Testserver ist **gut**."  

Da kommt nämlich irgendwie immer "gut" raus und wirkt bereits verbindlich, obwohl die Einschätzung möglicherweise nicht ganz  zutreffend sind. Das provoziert natürlich Rückfragen ("Internetverbindung ist doch gut! - Warum lädt mein Testheft nicht?")

Vermutlich wäre es irgendwo um diesen Abschnitt herum:

https://github.com/iqb-berlin/testcenter/blob/e4ff7d098f21714086210cfdda2c17a2c7c47de0/frontend/src/app/sys-check/network-check/network-check.component.html#L11

Alternativ könnte man auch die Werte verstellen, aber wir sind aus der Dokumentation nicht wirklich schlau geworden, was wo verstellt werden müsste, damit da realistischere Werte angegeben werden... 

```xml
<UploadSpeed min="1024" good="2048" maxDevianceBytesPerSecond="10000" maxErrorsPerSequence="0" maxSequenceRepetitions="15">100000, 200000, 400000, 800000</UploadSpeed>
<DownloadSpeed min="1024" good="2048" maxDevianceBytesPerSecond="200000" maxErrorsPerSequence="0" maxSequenceRepetitions="15">400000, 800000, 1600000, 3200000</DownloadSpeed>
```

Möglicherweise wären etwa Download 50 MBit und Upload 10 MBit plausibel. Allerdings kann ich das aktuell nicht sinnvoll hinterlegen (und dann auch noch einigermaßen plausible Abstufungen angeben).

**Comment by paflov** [comment]
min="1024" und good="2048" sind sehr schlechte Beispielwerte. Die Einheit ist auch hier BytesPerSecond - sehr viel höhere zahlen wären vermutlich sinnvoll. Hilft das weiter?

**Comment by franikowsp** [comment]
Genau, die sind zu gering. Aber was müsste ich denn einstellen, um Download 50 MBit und Upload 10 MBit zu erhalten? Mir fehlt etwas Verständnis von den Einheiten.

**Comment by paflov** [comment]
[50 Megabits/Second = 6250000 Bytes/Second](https://convertilo.com/50-megabits-second-to-bytes-second)
Ohne, dass ich es jetzt probiert habe - aber wenn du z. B. 50mBit/Sec minimal Wert für die Bewertung "gut" haben möchtest, müsstest du bei good 6250000 eingeben.
Sollte so funktionieren.

**Comment by franikowsp** [comment]
Danke Dir!

**Comment by paflov** [comment]
Ich mache daraus mal ein Ticket, das Wiki zu vervollständigen.
